packages/runner/src/traverse.ts

```diff
@@ -294,24 +395,37 @@ export abstract class BaseObjectTraverser<S extends BaseMemoryAddress> {
     if (isPrimitive(doc.value)) {
       return doc.value;
     } else if (Array.isArray(doc.value)) {
-      using t = tracker.include(doc.value, SchemaAll, doc);
+      const t = tracker.include(doc.value, SchemaAll, doc);
```

Comment:
Switching away from `using` means explicitly calling e.g. `[Symbol.dispose]()` -- could this continue to use that? It would dispose whenever `t` leaves scope. This uses the [explicit resource management](https://v8.dev/features/explicit-resource-management) feature in JS, supported in Deno, and polyfills used when needed in browsers. If automatic disposal isn't desired, consider a simple `dispose()` function instead